src/repository/index.ts

```diff
@@ -89,6 +95,7 @@ export default class Repository extends EventEmitter implements EventEmitter {
     instanceId,
     projectName,
     refreshInterval = 15_000,
+    maxRefreshInterval = 120_000,
```

Comment:
If we expose this in the constructor, it gives consumers the ability to just turn off our backoff. Do we want that? Tempted to use a constant here

Reply:
It's removed, and replaced by putting a maximum backoff of 10 * interval